### ISSUE TITLE
Updated Twitter 

### ISSUE THREAD

**Issue by spoorthyv** [jupyter/design#34]
I've been working on updating our twitter page. Specifically the profile picture and banner image. Let me know if you have any issues and opinions.

@ruv7

Profile Picture:
![frame 1](https://cloud.githubusercontent.com/assets/12418068/19404715/3fc7e582-9225-11e6-90c7-6ff7d0f0c731.png)

Banner Image:
![artboard 11](https://cloud.githubusercontent.com/assets/12418068/19404719/4826967e-9225-11e6-8b32-0330bc4bc317.png)

Preview:
![screen shot 2016-09-16 at 2 35 56 pm](https://cloud.githubusercontent.com/assets/12418068/19404722/4e07f2e0-9225-11e6-8348-1c5da4d26532.png)

Current Twitter:
![screen shot 2016-10-14 at 3 46 55 pm](https://cloud.githubusercontent.com/assets/12418068/19404784/9c0b7912-9225-11e6-97d4-c6a780d5344c.png)

**Comment by Ruv7** [comment]
Thanks Spoo! I'm on board with these changes and will look to see if anyone from the community has further feedback on changing our profile pic and this custom banner image. I see us using this across our social media pages which currently includes Twitter, Facebook, and LinkedIn. I also want to specifically flag @ellisonbg and @fperez  on this topic since this will be a highly visible update.

**Comment by ellisonbg** [comment]
Some comments:

* I like this a lot, works really well
* The actual twitter screenshot seems to show the banner in more faded colors than the above banner. Why is that? I prefer the less faded version in the banner.
* I am fine with the jupyter logo not having the word "jupyter" through the middle as the words "Project Jupyter" always appear next to it in Twitter.

**Comment by fperez** [comment]
Totally an improvement, thanks! Though while you're at it, if you can think of ways of going even further on the "less orange blast" for the main banner, that would be great.  The current one is a bit brutal in being blinding orange, and this change is a very nice improvement already.  I'm just wondering if some other changes could be added to lessen a bit further the "orange overload" and have the banner be a bit more subdued.

But I don't have any concrete suggestions, so if nothing else, +1 to this change. Thanks!!

**Comment by spoorthyv** [comment]
Here are a couple other options. I felt like the grey header made the page seem too dull, while having the white profile picture and banner made the page feel too empty. The second option here is probably my second favourite. I agree that the full orange banner is a bit overwhelming though.

![group](https://cloud.githubusercontent.com/assets/12418068/19500405/b5bac6d6-9551-11e6-9771-2fbce8a82fb2.png)

**Comment by spoorthyv** [comment]
Here is a version of Dark Grey with an orange chompy.

![artboard 13](https://cloud.githubusercontent.com/assets/12418068/19532329/165ba23c-95f0-11e6-91c7-742cc70ef239.png)

**Comment by Ruv7** [comment]
Hmmm. I'm not crazy about this one. I can't pinpoint why. The large amount of grey feels too dull. The more I look at the one with all the white space the more it grows on me. It is similar to our jupyter.org website. There is a large amount of white space in both designs...

**Comment by ellisonbg** [comment]
Can we try that last design with the jupyter text in pure white?

**Comment by spoorthyv** [comment]
Yeah, I think this is better.

![artboard 14](https://cloud.githubusercontent.com/assets/12418068/19574359/bfc6e77c-96be-11e6-8829-db35e97fe006.png)

**Comment by Ruv7** [comment]
I agree that looks better than the previous version but I would suggest we keep a consistent theme and feel in our marketing materials. The website should feel similar to the social media pages. The use of a large amount of grey is too big of a contrast. My vote is for the third design (white profile background/white banner background).

**Comment by Ruv7** [comment]
@spoorthyv @ellisonbg  - I'd like to pick this discussion back up so that we can come to a resolution this week. This has been sitting for a long time!

**Comment by spoorthyv** [comment]
After looking at this again, I kinda like the third one the most. I think it will have the most universal appeal. The grey and orange will have fairly polarizing opinions but I think the white is fairly neutral. Also it definitely matches our website perfectly.

**Comment by Ruv7** [comment]
Hey Spoo - Brian and I had a chance to review and discuss. His feedback was that given that these social media pages have alot of visual clutter going on in the interface that he felt it was important to contrast that with the darker design. Also he does see us beginning to use color blocking in our website and other materials. Therefore he would suggest we move forward with the design you posted on October 20. Let's review together next time you're in the office.

**Comment by cameronoelsen** [comment]
Here is a quick mockup of what the design that @spoorthyv posted on Oct 20 looks like on the twitter page. I like it!

![twitter-mockup](https://cloud.githubusercontent.com/assets/6437976/20991786/2f14b220-bc95-11e6-85a9-12c5e9fe4e4b.png)

**Comment by Ruv7** [comment]
Yup, I do too.

**Comment by Ruv7** [comment]
@spoorthyv - is that the final art file for insertion into the social media pages? Can you please upload to Dropbox and send me the link/confirm which folder you put this in?

**Comment by Ruv7** [comment]
@spoorthyv  and I were able to update 2 out of 3 pages without issue. The LinkedIn banner is not behaving as expected. Spoo is looking into a suggested update. @cameronoelsen @ellisonbg - take a look when you have a chance to let us know if all looks good.

Facebook: https://www.facebook.com/projectjupyter/

Twitter: https://twitter.com/ProjectJupyter

LinkedIn: https://www.linkedin.com/company/project-jupyter?report%2Esuccess=KJ_KkFGTDCfMt-A7wV3Fn9Yvgwr02Kd6AZHGx4bQCDiP6-2rfP2oxyVoEQiPrcAQ7Bf
